integration/confidential/lib.sh

```diff
@@ -222,6 +240,10 @@ setup_common_signature_files_in_guest() {
 }
 
 setup_offline_fs_kbc_signature_files_in_guest() {
+	# Enable signature verification via kata-configuration by removing the param that disables it
+	remove_kernel_param "agent.enable_signature_verification"
```

Comment:
`enable_signature_verification` disables signature verification?

Reply:
`enable_signature_verification` **enables** signatyre verification, if set to `true`, if set to `false` signature verification won't be enabled.

Reply:
Yeah this comment just struck me as a bit confusing.